content/about/introduction/aria-basics.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
My suggestion with this file is that it be given its own folder like the other about pages, that way the content structure is similar across all the about pages as well as across the pattern and practice pages.

Reply:
When I put aria-basics.html and introduction.html in the same directory, I was thinking of them both as part of introduction. However, I now see that one is introduction to APG while other is more introduction to ARIA. So, per your suggestion, I separated them.

In the future, there could be additional files or content under aria-basics that is related to the fundamentals of ARIA itself.